### PR TITLE
Indicate which postmaster items can't be pulled

### LINKED ISSUE #4715: Indicate which postmaster items can't be pulled.
Either dim them, or add some other visual indication that they can't be pulled from postmaster. See discussion in #4631 - highlighting pullable items on hover is another neat idea, but isn't mobile accessible (we could do this in *addition* to indicating the un-pullable ones to reinforce the connection with the button).

### PR DESCRIPTION
**Pull request by evan-willis**
Closes #4715

# Evidence
## New dimmed icons changes with items that can't be pulled
image

## With some blues that CAN be pulled
image

Note: First time trying to contribute to DIM so let me know how I'm doing!

### CODE REVIEW

File: src/app/inventory/DraggableInventoryItem.tsx [hunk 1]
```diff
@@ -19,6 +19,9 @@ export default function DraggableInventoryItem({ children, item }: Props) {
       ? item.equipment
       : item.equipment || item.bucket.hasTransferDestination;
 
+  const inPostmasterCantTransfer =
+    item.location.inPostmaster && (!item.canPullFromPostmaster || item.notransfer);
```
Comment (robojumper) [inline src/app/inventory/DraggableInventoryItem.tsx:23]: `notransfer` means "can't be transferred between characters/vault". It's probably not what you want/need.

Comment (evan-willis) [reply src/app/inventory/DraggableInventoryItem.tsx:23]: Removing the `item.notransfer` seems to show that I can still pull these items even when I don't have inventory space
image

As opposed to leaving it in
image

Comment (robojumper) [reply src/app/inventory/DraggableInventoryItem.tsx:23]: But it also means they'd be greyed out even when you do have the space :) 
Here's how other code determines this dynamic aspect https://github.com/DestinyItemManager/DIM/blob/6c2831c54d233ef0b4f0dc3f93fcb23c85a1101c/src/app/item-popup/item-popup-actions.ts#L72-L80

Comment (evan-willis) [reply src/app/inventory/DraggableInventoryItem.tsx:23]: Awesome I'll take a look 😃

**Comment by bhollis** [comment]
Hi @evan-willis please reopen whenever you get a chance to come back to this!

**Pull request closed without merging**